src/main/java/com/loofah/graph/api/resources/SkillsResource.java

```diff
@@ -0,0 +1,30 @@
+package com.loofah.graph.api.resources;
+
+import com.loofah.graph.api.services.SkillsService;
+import graphql.ExecutionResult;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RequestMapping("/skills")
+@RestController
+public class SkillsResource {
```

Comment:
Nit: how come you've gone for `SkillsResource` over `SkillsController` here? The latter feels more conventional for Spring Boot to me.

Reply:
I agree, I've updated to `SkillsController`